elastic/kibana
# Enable search bar/timepicker for logs only services

## Summary

Enables logs only Services to have a timepicker in the logs tab.

![Screenshot 2024-10-31 181045](https://github.com/user-attachments/assets/0026af42-313c-4aa5-82b3-5f726d3e52d0)

### How to test

- Use `node scripts/synthtrace traces_logs_entities.ts --clean --live`
- Navigate to `synth-go-logs` service, go to logs tab
- Timepicker should be visible and work to filter available logs

Closes #197604
